__test__/FilterSidebarCheckboxFacetGroup.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import Enzyme from 'enzyme'
+import {mount} from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+import _ from 'lodash'
+
+import {getRandomInt, episodes} from './TestUtils'
+
+import FilterSidebar from '../src/FilterSidebar'
+import CheckboxFacetGroup from '../src/facetgroups/CheckboxFacetGroup'
+
+Enzyme.configure({ adapter: new Adapter() })
+
+describe(`FilterSidebarCheckboxDropdownFacetGroup`, () => {
+  const allFacets = episodes.reduce((acc, episode) => acc.concat(episode.facets), [])
+  const uniqueFacets =
+    allFacets
+      .filter((facet, index) => allFacets.findIndex((thatFacet) => facet.value === thatFacet.value) === index)
+      .map((facet) => ({
+        ...facet,
+        disabled: false
+      }))
```

Comment:
You’re not using `allFacets` nor `uniqueFacets` in your tests. Either remove them and keep your hard-coded values or pick a random one from here (I’d rather do the latter, always try to randomize test data, so that the test logic focuses on the invariants in your code).